File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:

        nums = set(nums)
        if len(nums) == 0:
            return 0
            
        max_consecutive = 1
        for num in nums:
            if num - 1 in nums:
                continue
            else:
                temp = 0
                while num in nums:
                    temp+=1
                    num+=1
            max_consecutive = max(temp, max_consecutive)
        return max_consecutive
        

File: Data Structures & Algorithms/duplicate-integer/submission-1.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        map<int,int> duplicates;

        for(int num: nums) {
            duplicates[num]+=1;
            if(duplicates[num]>1) {
                return true;
            }
        }
        return false;
    }
};